### DISCUSSION TITLE
modcharts.

### DISCUSSION THREAD

**Discussion by libracrumbs** [KadeArchive/Kade-Engine#2726] [Q&A]
hi, so i dont know how modcharts work and at wiki it just shows the functions, how to change note positions? like in qt mod

**Comment by JustinSolstice** [comment]
You can use tweenPos to change note positions smoothly 

ex. 

` for i = 4, 7 do
        tweenPos(i, _G['defaultStrum'..i..'X'] - 275,G['defaultStrum'..i..'Y'], 0.5, 'tweenfornotes')
end`

 --this makes the notes go to the center
